UNRELEASED.md

```diff
@@ -7,3 +7,9 @@
 **Fixed:**
 - bpk-component-badge:
   - Added a default font colour to badges to prevent the badge colour changing due to cascading styles.
+
+**Breaking:**
+- bpk-mixins:
+    - Added `border-color` to input fields to highlight red when there is an error on the input
```

Comment:
Could all of these three be more specific about which state of the input and which mixins specifically that is changing? Can we also end each line with a period?

Reply:
Also `bpk-component-input` should be listed under this breaking change.

Reply:
And I guess also `bpk-component-fieldset`? 🤔